pyro/PackageManager.py

```diff
@@ -184,7 +184,7 @@ def create_packages(self) -> None:
                 os.makedirs(os.path.dirname(target_path), exist_ok=True)
                 shutil.copy2(source_path, target_path)
 
-            compress: bool = package_node.get('Compress')
+            compress: bool = package_node.get('Compress') == 'True'

```

Comment:
Here I found a bug I made, that caused Compress="False" to enable compression.
It's fixed now, I copied this line from here: https://github.com/fireundubh/pyro/blob/f4fd061609e8db44c7c51014240a6220a64e500d/pyro/PapyrusProject.py#L96

But this means that only Compress="True" evaluates to True, even though the XML schema suggests that true, tRue, and 1 should also work: https://github.com/fireundubh/pyro/blob/f4fd061609e8db44c7c51014240a6220a64e500d/pyro/PapyrusProject.xsd#L115

Is this a bug or am I confused?

Reply:
> Is this a bug or am I confused?

The first two patterns in the XSD allow users to enter `true` or `false` case insensitively.

The third pattern allows `1` and `0` whose meanings are handled in Python.

`PapyrusProject._update_attributes` cleans up, formats, and/or defaults relevant attribute values, and `str(True) == 'True'` in Python.

By the time we call `self.ppj_root.get('Optimize')`, the value is either `'True'` or `'False'` and cannot be anything else.

Reply:
Okay, now it makes sense :D. Wasn't working for compress because I hadn't added it to _update_attributes.